lib/types.nix

```diff
@@ -406,7 +406,7 @@ rec {
                 In file ${def.file}
                 a list is being assigned to the option config.${option}.
                 This will soon be an error as type loaOf is deprecated.
-                See https://git.io/fj2zm for more information.
+                See https://github.com/NixOS/nixpkgs/pull/63103 for more information.
```

Comment:
Should we wrap this at `more`?

Reply:
I thought about it, but decided instead to go with the idea that the original author intended there to be one full sentence per line. (The templated bits in the displayed message often greatly exceed the line length of the line I added.)